### PR TITLE
[APP] Add tensorflow based product rating

### PR DESCRIPTION
**Pull request by kparichay**
Added tensorflow based product rating.

**Self evaluation:**
1. Build test: [x]Passed [ ]Failed [ ]Skipped
2. Run test: [x]Passed [ ]Failed [ ]Skipped

Signed-off-by: Parichay Kapoor <pk.kapoor@samsung.com>

### CODE REVIEW

File: Applications/ProductRatings/jni/main.cpp [hunk 1]
```diff
@@ -7,7 +7,7 @@
  * @see    https://github.com/nnstreamer/nntrainer
  * @author Jijoong Moon <jijoong.moon@samsung.com>
  * @bug    No known bugs except for NYI items
- * @brief  This is Binary Logistic Regression Example
+ * @brief  This is a simple recommentation system Example
```
Comment (zhoonit) [inline Applications/ProductRatings/jni/main.cpp:10]:
```suggestion
 * @brief  This is a simple recommendation system Example
```

**Review by zhoonit — APPROVED** [review]
LGTM

File: Applications/ProductRatings/jni/main.cpp (hunk 1 above)
Comment (kparichay) [reply Applications/ProductRatings/jni/main.cpp:10]: done.

**Review by lhs8928 — APPROVED** [review]
LGTM

File: Applications/ProductRatings/Tensorflow/product_ratings.py [hunk 2]
```diff
@@ -0,0 +1,47 @@
+#!/usr/bin/env python3
+# SPDX-License-Identifier: Apache-2.0
+#
+# Copyright (C) 2021 Parichay Kapoor <pk.kapoor@samsung.com>
+#
+# @file	  embedding.py
```
[the remark this answers is not in the document]
Comment (kparichay) [reply Applications/ProductRatings/Tensorflow/product_ratings.py:6]: done.

**Review by jijoongmoon — APPROVED** [review]
LGTM

**Pull request merged**